js/freewheel.js

```diff
@@ -962,6 +977,16 @@ OO.Ads.manager(function(_, $) {
     }, this);
 
     /**
+     * Checks to see if we can use a fake video to trick Freewheel into showing overlays
+     * in a div that is intregrated with Alice.
+     * @private
+     * @method Freewheel#_supportsSecondaryVideoForOverlay
+     */
+    var _supportsSecondaryVideoForOverlay = _.bind(function() {
```

Comment:
Is it necessary to put this into a function? Just wondering, since it is technically a function calling a function.